# HG changeset patch
# User Mark Lodato <lodatom@gmail.com>
# Date 1254700923 14400
freeze: remove blank lines from beginning and end


diff --git a/bin/cython_freeze b/bin/cython_freeze
index b9441e4..1bb582a 100755
--- a/bin/cython_freeze
+++ b/bin/cython_freeze
@@ -34,7 +34,7 @@ def format_modname(name):
 
 modules = [format_modname(x) for x in args]
 
-print """
+print """\
 #include <Python.h>
 #include <stdio.h>
 #include <stdlib.h>
@@ -86,4 +86,4 @@ int wmain(int argc, wchar_t **argv) {
     Py_Finalize();
     return r;
 }
-""" % {'main' : modules[0]}
+""" % {'main' : modules[0]},
-- 
1.6.5.rc3

